docs/source/install_instructions.rst

```diff
@@ -34,26 +34,26 @@ How to setup OOIPY development environment
 
 .. code-block :: bash
 
-  conda create --name env_name
+  conda create --name env_name pip
 
 3. Activate the environment using this command:
 
 .. code-block :: bash
 
   source activate env_name
 
-4. Run the script to setup the development environment and install dependencies using this command:
+4. Develop ooipy using this command in the same directory where the repository has been cloned:
```

Comment:
I think we can change this to:
4. install ooipy using this command in the same directory where the repository has been cloned.

Reply:
Done